### ISSUE TITLE
Feature wish: Threaded view for hashtag searching

### ISSUE THREAD

**Issue by rabuzarus** [friendica/friendica#6689]
### Expected behavior
Friendica should have a possibility to displayed mesages filtered by a hashtag in a threaded view (with comments)

### Actual behavior
At the moment Friendica can only search for tagged messages through the search module and the result will presented item by item. The see the whole thread you need to click on 'view in context'

**Comment by annando** [comment]
Currently we do search in thread starting posts and in comments. When enabling this feature, we could only search in thread starting posts. Otherwise you had the problem that you would get a whole thread, just because this hashtag had been used somewhere in the whole thread - possibly in a single comment.

**Comment by rabuzarus** [comment]
yes I know. The question is also if the search module is the correct place or if we introduce a new `tag` module for threaded view. I just think we should have such a feature. Thinking of a live updated threaded view with `#noG20` ;-)

**Comment by annando** [comment]
This live updated threaded view already does work - when you had added the hashtag into the saved searches.

**Comment by rabuzarus** [comment]
the saved search will link you to `/search?search=%23nog20` the search module doesn't have a threaded view

**Comment by annando** [comment]
Yes. But the "network" will get live posts with this hash tag when you add it to your saved searches.

**Comment by rabuzarus** [comment]
Yes I know but:
1.) there are all the thousands of other posts which I don't want to read
2.) I don't wan't to subscribe to every hastag which I want to read.
